src/main/java/uk/gov/hmcts/reform/notificationservice/model/out/NotificationsResponse.java

```diff
@@ -34,7 +35,9 @@
     public final List<NotificationInfo> notifications;
 
     public NotificationsResponse(List<NotificationInfo> notifications) {
-        this.notifications = notifications;
+        this.notifications = notifications.stream()
+            .sorted(Comparator.comparingLong(i -> Long.parseLong(i.id)))
```

Comment:
I think you do not need to function . sorted() would be enough.

Reply:
We need the comparator as sorted only method throws an exception. I think that is because NotificationInfo class does not define comparable interface or so. I have tested it.

Reply:
I would suggest to do ordering on SQL level, i.e. in `NotificationRepository` class.

Reply:
I would do that Andrey. As always, I am curious what would be the advantage? I know you have more experience and you can tell us why you would prefer doing the sorting at the query level.